.github/workflows/ci-new.yml

```diff
@@ -165,6 +165,12 @@ jobs:
           service_account: '${{ env.WIF_SERVICE_ACCOUNT }}'
       - name: 'Setup gcloud'
         uses: 'google-github-actions/setup-gcloud@e30db14379863a8c79331b04a9969f4c1e225e0b' # ratchet:google-github-actions/setup-gcloud@v1
+      - id: 'create_kms_keyring'
+        name: 'Create kms keyring'
+        run: |-
+          gcloud kms keyrings create ${{ env.KMS_KEYRING_ID }} \
+            --project ${{ env.PROJECT_ID }} \
+            --location global
```

Comment:
Hey @sailorlqh - can you share more info about why we need to create a new keyring on each CI run? It seems like we should be able to create a cryptokey and N cryptokeyversions for each CI run instead.

Reply:
The current way is create a new ring in the same keyring. The reason for this is discussed in the infra repo: https://github.com/abcxyz/infra/pull/552#discussion_r1352873887